.vib/odoo/cypress/cypress/integration/odoo_spec.js

```diff
@@ -10,6 +10,13 @@ it('allows installing/uninstalling an application and inviting new users', () =>
   });
   cy.reload();
 
+  // Perform a second login in case the user is logged out after installing the Sales app
+  cy.get('#login', {timeout: 0}).then($login => {
+    if ($login.is(':visible')) {
```

Comment:
I understand the `is` assertion is similar to `should`, but I actually couldn't find the Cypress documentation for it, could you please share it?

Reply:
I read it in some forum :)

Here for example:

https://testup.io/how-to-check-if-an-element-exists-or-not-using-cypress-io/

Reply:
It's actually a Jquery method that Cypress uses under the hood